.flake8

```diff
@@ -7,6 +7,7 @@
 
 # Some rules are considered irrelevant to stub files:
 #     E701   multiple statements on one line (colon) -- disallows "..." on the same line
+#     E704   multiple statements on one line (def) -- disallows "..." on the same line
```

Comment:
Why was this change needed? Seems unrelated to adding new stubs.

Reply:
I figured out later what caused E704 to show up in my editor. Should be removed